loops-and-arrays.js

```diff
@@ -23,6 +30,20 @@ const names = ["Henk", "Piet", "Fred", "Joop"];
 
 const numbers = [2, 4, 5, 29, 38];
 
+console.log(numbers);
+
+for (let i = 0; i < numbers.length; i++){
+    if (numbers[i] % 2 === 0 ){
+        numbers[i] = numbers[i] * 2;
+    }
+    if(numbers[i] % 2 !== 0) {
```

Comment:
Je hebt hier 2 if statements. Dit is redundant (niet nodig), omdat je ook volstaat met een if / else statement ipv een if / if.
Dan hoef je die 2e if niet te specificeren omdat die toch al geldt als rest data (dus else)... Ik hoop dat ik het duidelijk vertel en goed heb en mijn logica klopt... hahaha.

Maarrrrrrrr... wat praat ik nou als je het precies als het antwoordmodel hebt gedaan!!!! Dus prima gedaan Jelmer.

Reply:
Je hebt helemaal gelijk! Jouw manier is inderdaad sneller!